docs/geedocs/answer/3468016.html

```diff
@@ -1,16 +1,16 @@
 <!DOCTYPE html> <html> <head> <meta charset="UTF-8"/>
-<title>Download GEE 5.x</title>
+<title>Build and install GEE 5.x</title>
 <link rel="stylesheet" href="../css/styles.css" type="text/css" media="all" /> <link rel="stylesheet" href="../css/containers.css" type="text/css" media="all" /> </head> <body>
 <!-- 3468016.html -->
 <div class="container">
   <div class="sidebar1">
 <p class="sidebar-toc"><a href="../answer/2995465.html">Welcome to Google Earth Enterprise</a></p>
 <p class="sidebar-toc"><a href="../answer/6353404.html">Overview</a></p>
-<p class="sidebar-toc"><a href="../answer/3468016.html" class="current-file">Download GEE 5.x</a></p>
+<p class="sidebar-toc"><a href="../answer/3468016.html" class="current-file">Build and install GEE 5.x</a></p>
 <p class="sidebar-toc"><a href="../answer/3477797.html">Upgrade to GEE 5.x</a></p>
 <p class="sidebar-toc"><a href="../answer/6025486.html">Download documentation for GEE 5.1.x</a></p>
 <p class="sidebar-toc"><a href="../answer/2760566.html">Installation tips</a></p>
-<p class="sidebar-toc"><a href="../answer/4627891.html">Supported Versions of Fusion, GEE Server, and Portable Server</a></p>  </div>
+</div>
   <div class="content">
 <a name="top_of_file"></a>
 <p><img src="../art/common/googlelogo_color_260x88dp.png" width="130" height="44" alt="Google logo" /></p>
```

Comment:
The two are still logically separate, and they can be installed separately.  I guess they're not technically "packages", but we hope they will be soon.  I decided this was accurate enough that it didn't need to change.